### PR TITLE
Structural/Adding the d+d- and isotropic damage CL in 2D

### PR DESCRIPTION
**Pull request by AlejandroCornejo**
In this PR I'm adding the d+d- and isotropic damage laws for the 2D case. Besides I've improved some fatures of the d+d- cl that some people asked for.

### CODE REVIEW

**Comment by AlejandroCornejo** [comment]
ping

**Comment by AlejandroCornejo** [comment]
@loumalouomega  pls take a look whenever you can :)

File: applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_d_plus_d_minus_damage.cpp [hunk 1]
```diff
@@ -457,7 +456,7 @@ double& GenericSmallStrainDplusDminusDamage<TConstLawIntegratorTensionType, TCon
     } else if (rThisVariable == UNIAXIAL_STRESS_TENSION) {
        rValue = mTensionUniaxialStress;
     } else {
-       return BaseType::GetValue(rThisVariable, rValue);
+       //return BaseType::GetValue(rThisVariable, rValue);
```
Comment (loumalouomega) [inline applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_d_plus_d_minus_damage.cpp]: Why commenting?

**Review by loumalouomega — CHANGES_REQUESTED** [review]
Look at my comment

File: applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_d_plus_d_minus_damage.cpp (hunk 1 above)
Comment (AlejandroCornejo) [reply applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_d_plus_d_minus_damage.cpp]: I forgot to uncomment jejeje thank you

**Review by loumalouomega — APPROVED** [review]
I activate it myself, I approve

**Pull request merged**